www/tilesource-iris.html

```diff
@@ -0,0 +1,166 @@
+<h2>
+    Example: Iris TileSource Support
+</h2>
+<p>
+    The <strong>IrisTileSource</strong> is an OpenSeadragon tilesource implementation for viewing images served by an Iris-compatible tile server. It is designed to efficiently display large, high-resolution images by loading only the visible tiles at the appropriate zoom level.
+</p>
+<p>
+    The Iris API provides endpoints for retrieving image metadata and individual image tiles. OpenSeadragon's IrisTileSource fetches the metadata to determine the image's dimensions, available zoom levels, and tile layout, then requests tiles as needed while you pan and zoom.
+</p>
+
+<div class="description">
+    <h3>IrisTileSource Configuration</h3>
+    <p>
+        To use the IrisTileSource, specify the <code>type</code> as <code>IrisTileSource</code> and provide the <code>serverUrl</code> (the base URL of your Iris server) and <code>slideId</code> (the image identifier).
+    </p>
+</div>
+<div class="demoarea">
+  <div class="demoheading">
+      Example Inline Configuration for Custom Tile Sources
```

Comment:
Just noting that this isn't a real demo yet. Of course I understand it is a work in progress.

Reply:
Excellent.